src/orders/components/OrderManualTransationRefundPage/components/OrderManualTransationRefundTiles/OrderManualTransationRefundTiles.tsx

```diff
@@ -55,17 +48,45 @@ export const OrderManualTransationRefundTiles = ({
             {transactions.map(transaction => (
               <OrderTransactionTile error={!!error} key={transaction.id}>
                 <OrderTransactionTile.Header>
-                  <RadioGroup.Item
-                    {...field}
-                    id={transaction.id}
-                    value={transaction.id}
-                    error={!!error}
-                    padding={4}
-                  >
-                    <Text size={5} fontWeight="medium" padding={4}>
-                      {transaction?.name || "Transaction"}
-                    </Text>
-                  </RadioGroup.Item>
+                  {isRefundable(transaction) ? (
+                    <RadioGroup.Item
+                      {...field}
+                      id={transaction.id}
+                      value={transaction.id}
+                      error={!!error}
+                      padding={4}
+                    >
+                      <Text size={5} fontWeight="medium" padding={4}>
+                        {transaction?.name || "Transaction"}
+                      </Text>
+                    </RadioGroup.Item>
+                  ) : (
+                    <Tooltip>
+                      <Tooltip.Trigger>
+                        <RadioGroup.Item
+                          {...field}
+                          id={transaction.id}
+                          value={transaction.id}
+                          disabled
+                          error={!!error}
+                          padding={4}
+                        >
+                          <Text
+                            size={5}
+                            fontWeight="medium"
+                            padding={4}
+                            color="defaultDisabled"
+                          >
+                            {transaction?.name || "Transaction"}
+                          </Text>
+                        </RadioGroup.Item>
+                      </Tooltip.Trigger>
+                      <Tooltip.Content side="left">
+                        <Tooltip.Arrow />
+                        <FormattedMessage {...messages.notRefundable} />
+                      </Tooltip.Content>
+                    </Tooltip>
+                  )}
```

Comment:
**suggestion**: I really dislike this pattern, showing a tooltip on disabled component is very common use case, I believe this should be fixed in macaw-ui by adding `disabled` prop to tooltip - which if `true` would just render the children skipping the tooltip altogether.